examples/AccountManagement/CreateCustomer.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Ads\GoogleAds\Examples\AccountManagement;
+
+require __DIR__ . '/../../vendor/autoload.php';
+
+use GetOpt\GetOpt;
+use Google\Ads\GoogleAds\Examples\Utils\ArgumentNames;
+use Google\Ads\GoogleAds\Examples\Utils\ArgumentParser;
+use Google\Ads\GoogleAds\Lib\GoogleAdsClient;
+use Google\Ads\GoogleAds\Lib\GoogleAdsClientBuilder;
+use Google\Ads\GoogleAds\Lib\GoogleAdsException;
+use Google\Ads\GoogleAds\Lib\OAuth2TokenBuilder;
+use Google\Ads\GoogleAds\Util\ResourceNames;
+use Google\Ads\GoogleAds\V0\Errors\GoogleAdsError;
+use Google\Ads\GoogleAds\V0\Resources\Customer;
+use Google\ApiCore\ApiException;
+use Google\Protobuf\BoolValue;
+use Google\Protobuf\StringValue;
+
+/**
+ * This example illustrates how to create a new customer under a given manager account.
+ *
+ * Note: this example must be run using the credentials of a Google Ads manager account. By default,
+ * the new account will only be accessible via the manager account.
+ */
+class CreateCustomer
+{
+    const CUSTOMER_ID = 'INSERT_CUSTOMER_ID_HERE';
```

Comment:
Shouldn't you use a different constant, since this needs to be a manager ID specifically? Also a little confusing to require a customer ID to create a customer.

Reply:
Thanks for catching this.
Actually the ID shouldn't be specified in the new Customer object too. I'll point this out in the golden PR.